backend/VERSION.json

```diff
@@ -1,4 +1,4 @@
-{ "VERSION" : "4.15.0",
+{ "VERSION" : "4.14.5",
```

Comment:
? it doesn't look like you based off master, but the version number is altered?

Reply:
Hmm I don't know what to make of that! My last commit integrated changes from the branch holding the 4.14.5 tag made at the end of March. I always pull the latest version before making the PR, so the only thing I can think of is I accidentally merged master instead of staging!? I will revert whatever it was either way and fix it...

Reply:
@carmenfan OK it should be fixed now.